uiowa/uiowa
# Update users:list command to get non-admins who have logged in recently.

First step to address #3153. We'll need to deploy this to actually run it on all sites and then do some copy/paste for a report. We'll need to add a similar option to D7 `herky-list-users` command .


### Testing
- `blt sync --site adsforruralamerica.uiowa.edu`
- `drush -l adsforruralamerica.uiowa.edu users:list --no-roles administrator --last-login '9 months ago'`
- See that no users are found meaning the site is "inactive" according to our definition.
- Double check that no non-admins have logged in to the site within 9 months:
  - `drush -l adsforruralamerica.uiowa.edu users:list`
- `blt sync --site uiowa.edu`
- `drush -l uiowa.edu users:list --no-roles administrator --last-login '9 months ago'`
- See that non-admins have logged in recently.
